manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -491,3 +491,7 @@ defaults:
   istio_cni:
     enabled: false
     chained: true
+  # applies to mutating and validation webhook
+  webhookConfig:
```

Comment:
I think this won't work, the webhook caBundle will be patched by Istiod's certs.

Reply:
ohh this works fine if you disable the istio patching with env variable. We have been running Istio this way but we had to patch using kustomize to support override caBundle. This PR will help us get rid of kustomize patches

Reply:
For context, we have been running Istio with well known cert and Google CAS for workload certificate

Reply:
> if you disable the istio patching with env variable.

This makes sense to me, thanks